src/components/10-atoms/button/index.js

```diff
@@ -28,6 +30,12 @@ class AXAButton extends LitElement {
     `;
   }
 
+  // Parent class InlineStyles needs a static method to retrive styles
+  // name of such method is passed when calling: this.inlineStyles('resetHeadingCss');
+  static get resetHeadingCss() {
```

Comment:
@NoahLiechti Please rename this as it does not reset headings rather is blocks mouse events or you can also call it direct child styles